src/ucp/api/ucp.h

```diff
@@ -1201,7 +1203,7 @@ struct ucp_tag_recv_info {
  * @ingroup UCP_CONTEXT
  * @brief Operation parameters passed to @ref ucp_tag_send_nbx,
  *        @ref ucp_tag_send_sync_nbx, @ref ucp_tag_recv_nbx, @ref ucp_put_nbx,
- *        @ref ucp_get_nbx
+ *        @ref ucp_get_nbx, @ref ucp_am_send_nbx and @ref ucp_am_recv_data_nbx.
```

Comment:
do there functions exist in this branch?

Reply:
> do there functions exist in this branch?

some of funds exist, but return NOT_IMPLEMENTED, cleaned comments